html/Axit/js/validation/helper-functions.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable max-len */
+export function checkName(nameInput, error) {
+  const localError = error;
+  if (
+    !isInputNotEmpty(nameInput, error, 'Name is required') ||
+    !isNameCorrect(nameInput, error, 'Only latin letters')
+  ) {
+    return false;
+  }
+  localError.innerHTML = '';
```

Comment:
check function(and validation function for Name/Email/etc) should only do one thing, in this case check value, not also add errors and work with dom, same below